bundles/org.openhab.binding.gree/README.md

```diff
@@ -41,7 +41,7 @@ The following channels are supported for fans:
 |                    |           | Mode can be one of auto/cool/eco/dry/fan/heat or on/off                                           |
 |                    |           | Check the Air Conditioner's operating manual for supported modes.                                 |
 | temperature        | Number:Temperature | Sets the desired room temperature.                                                       |
-| currentTemperature | Number:Temperature | Displays the current room temperature.                                                   |
+| currentTemperature | Number:Temperature | Displays the current room temperature (if supported by unit, otherwise NaN).                                                   |
```

Comment:
```suggestion
| currentTemperature | Number:Temperature | Displays the current room temperature (if supported by unit, otherwise NaN).               |
```

Reply:
chaged